### DISCUSSION TITLE
How to fire an event manually in version 7, like what we have in old version with fireEvent() method

### DISCUSSION THREAD

**Discussion by Idrus82** [katspaugh/wavesurfer.js#3694] [Q&A] [answered]
In the old version (6), we have a fireEvent method to fire an event manually.

`wavesurfer.on('ready', function () {
    wavesurfer.fireEvent('play');
});`

I get an error using that method in version 7
`...fireEvent is not a function`.

How to fire an event manually in version 7?

**Comment by katspaugh** [comment, marked as answer]
Events aren't supposed to be fired manually. If you want to trigger playback, you have to call the `play()` method.
